cql/ManageCommonAbnormality.cql

```diff
@@ -365,8 +372,84 @@ define PositivePrimaryHpv:
 define MostRecentCytologyCotestWasAfterAge21:
   Collate.MostRecentCytologyCotest.date >= Patient.birthDate + 21 years
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+// Guidance for expedited Treatment (rarely screened) (Box 1. Essential Changes from Prior Management Guidelines, #3)
+// Expedited treatment is preferred for never or rarely screened patients with HPV-positive HSIL cytology regardless of HPV genotype.
+
+define RarelyScreenedLookbackInterval:
+  Interval[Collate.MostRecentCytologyCotest.date - Collate.RarelyScreenedInterval, Collate.MostRecentCytologyCotest.date)  
+  
+
+define IsHsilAndHpvPositive:
+  Collate.MostRecentCytologyCotest is not null and
+  Collate.MostRecentCytologyCotestResult = 'HSIL+' and
+  HighRiskHPVPositiveResults
+  
+define HighRiskHPVPositiveResults:
+  AnyTrue(
+    (Collate.MostRecentHpvReport.allConclusions) aC 
+      return aC in Dash."High Risk HPV Positive Results"
+  )
+  
+define PreviousCytologyOrSurveillanceIsOldOrAbsent:
```

Comment:
Instead of checking if:
```
    No Testing exists OR
    Testing exists but it is outside of the RarelyScreenedLooback interval
```
it may be simpler just to check that:
```
    No testing exists within the RarelyScreened Lookback interval
```
I know we had a conversation around how to structure this logic. Was there a reason we formatted it this way?

Reply:
Oh, I see now - yes, you can remove the first clause checking for no testing exists (395-399). It is logically redundant

Reply:
Logic was re-worked in the most recent commit: https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/115/commits/97bfa9d1323caa0e7aa2d414dbf3a57efdef635a. Looks good to me!